overrides/default.nix

```diff
@@ -1886,7 +1886,12 @@ lib.composeManyExtensions [
 
         # For OSX, we need to add a dependency on libcxx, which provides
         # `complex.h` and other libraries that pandas depends on to build.
-        postPatch = lib.optionalString (!(old.src.isWheel or false) && stdenv.isDarwin) ''
+        postPatch = ''
+          if [ -f pyproject.toml ]; then
```

Comment:
Rather than checking for the presence of a file in Bash I think it's preferable to check for the package version in the Nix code. Doing it that way means we can remove the check once the older version is definitely no longer relevant.

Reply:
Its been a while since I've looked at this PR, afaik I did it this way to mimic the existing `setup.py` patch.

Regarding versions the problematic dependency is still the same on the primary development branch, so it needs to be applied to for all versions:
https://github.com/pandas-dev/pandas/blob/4921e0b2290f881e17bd21685289e46ae4894155/pyproject.toml#L5

Reply:
This seems fine for now.